tests/cms/views/organizations/test_organization_form.py

```diff
@@ -141,3 +140,169 @@ def test_organization_form_shows_associated_contents(
 
     else:
         assert response.status_code == 403
+
+
+@pytest.mark.django_db
+def test_create_new_organization(
+    load_test_data: None,
+    login_role_user: tuple[Client, str],
+    settings: SettingsWrapper,
+    caplog: LogCaptureFixture,
+) -> None:
+    """
+    Test an organization will be created as expected
+    """
+    client, role = login_role_user
+
+    settings.LANGUAGE_CODE = "en"
+
+    new_organization = reverse(
+        "new_organization",
+        kwargs={
+            "region_slug": REGION_SLUG,
+        },
+    )
+    response = client.post(
+        new_organization,
+        data={
+            "name": "New Organization",
+            "slug": "new-organization",
+            "icon": 1,
+            "website": "https://integreat-app.de/",
+        },
+    )
+
+    if role in HIGH_PRIV_STAFF_ROLES + [MANAGEMENT]:
+        assert response.status_code == 302
+        assert_message_in_log(
+            'SUCCESS  Organization "New Organization" was successfully created',
+            caplog,
+        )
+        edit_url = response.headers.get("location")
+        response = client.get(edit_url)
+        assert (
+            "Organization &quot;New Organization&quot; was successfully created"
+            in response.content.decode("utf-8")
+        )
+        assert Organization.objects.filter(name="New Organization").exists()
+
+    elif role == ANONYMOUS:
+        assert response.status_code == 302
+        assert (
+            response.headers.get("location")
+            == f"{settings.LOGIN_URL}?next={new_organization}"
+        )
+    else:
+        assert response.status_code == 403
+
+
+@pytest.mark.django_db
+def test_cannot_create_organization(
```

Comment:
I think I would be a bit more specific in the title of the test and call it maybe something like "test_cannot_create_organization_with_duplicate_name". It is very long, but I think being explicit is more desirable